app.rb

```diff
@@ -8,14 +8,23 @@
 require_relative './lib/ui/io.rb'
 
 def initializer(size, token1, token2)
+  human_script = {
+    get_move: "\nPlease enter a number between 1 - 9: ",
+    announce_move: "\nYou picked spot "
+  }
+  ai_script = {
+    get_move: "\nThe computer is picking a spot...",
+    announce_move: "\nThe computer picks spot "
+  }
+
```

Comment:
Can you make a PlayerScript struct or class, to give an interface to these scripts? [] accessors can be a little confusing since there's nothing obvious about their properties